src/System.IO.Compression/tests/Configurations.props

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netstandard1.3-Unix;
+      netstandard1.3-Windows_NT;
+      netstandard1.7-Unix;
+      netstandard1.7-Windows_NT;
+      netstandard1.7;
```

Comment:
I think I've identified that this is happening from .builds files that include a Performance test project, because it's usually included with default targets. I'm pretty sure all the files you mentioned this on have an extra configuration because of that.